.travis.yml

```diff
@@ -10,10 +10,10 @@ env:
   - GO111MODULE=on GOFLAGS=-mod=readonly
 
 before_install:
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.15.0
+  - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.19.1/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.19.1
```

Comment:
FWIW

@kamilogorek this bump introduces new linters that would cause the build to fail 😞 
I wouldn't want to please linters right now, some are even questionable. So the solution would be to make future code more strict by using the `--new-from-rev` flag.

I also had to give up on v1.20.0 and v1.21.0 as detailed in the commit message.

Reply:
Makes sense, thanks for updating that